### ISSUE TITLE
SETI Rebalance's Hybrid Rocket Booster errors when Procedural Parts is installed

### ISSUE THREAD

**Issue by ghost** [Kerbas-ad-astra/SMURFF#4]
```

[LOG 12:42:18.884] [ModuleManager] Applying node SMURFF/SMURFF/@PART[*]:HAS[@MODULE[ModuleEngines*],!MODULE[ModuleEngines*]:HAS[!PROPELLANT[Oxidizer]],!MODULE[ModuleCommand],~CrewCapacity[>0],!MODULE[ModuleCargoBay],@MODULE[ModuleSMURFF],~SMURFFExclude[*rue]]:FOR[zzz_SMURFF] to SETIrebalance/0ProceduralParts/SETI-PP-HRB/proceduralTankHRB

[LOG 12:42:18.885] [ModuleManager] Cannot find key reservedmass in PART

[LOG 12:42:18.886] [ModuleManager] Error - Cannot parse variable search when editing key mass = #$reservedmass$

[LOG 12:42:18.889] [ModuleManager] Cannot find key reservedmass in PART

[LOG 12:42:18.890] [ModuleManager] Error - Cannot parse variable search when editing key mass = #$reservedmass$

[LOG 12:42:18.896] [ModuleManager] Applying node SMURFF/SMURFF/@PART[*]:HAS[@MODULE[ModuleEngines*],!MODULE[ModuleEngines*]:HAS[!PROPELLANT[Oxidizer]],!MODULE[ModuleCommand],~CrewCapacity[>0],!MODULE[ModuleCargoBay],@MODULE[ModuleSMURFF],~SMURFFExclude[*rue]]:FOR[zzz_SMURFF] to SETIrebalance/MM-NewParts/hybridBoosterRT-11/hybridBooster-RT11/hybridBooster
```

uhh

error goes away when deleting the files related to the Hybrid Rocket Booster

**Comment by Kerbas-ad-astra** [comment]
Resolved in 2c0674a.  Sorry it took so long to inform you!

**Comment by ghost** [comment]
Thanks!
